README.md

```diff
@@ -11,7 +11,7 @@ write me
 
 Translation Status
 ------------------
-[![Transifex Status](https://www.transifex.com/projects/p/OCA-l10n-mexico-12-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-l10n-mexico-12-0)
+[![Transifex Status](https://www.transifex.com/projects/p/OCA-l10n-mexico-13-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-l10n-mexico-13-0)
```

Comment:
@Dangar-Serpentcs You can replace this Transifex link with the Weblate one. See the example on OCA/field-service.